ci/build.sh

```diff
@@ -54,13 +54,17 @@ fi
 echo "version: $jdk_version"
 
 echo "Adding openjdk to bosh blobs"
-#jdk_blob_filename="jdk-${jdk_version}.tar.gz"
-jre_blob_filename="jre-${jdk_version}.tar.gz"
+jdk_blob_filename="jdk-${jdk_version}.tar.gz"
+jre_blob_filename="jre-${jre_version}.tar.gz"
+
+
+if compare_blob_sha_with_new_file ${jdk_blob_filename} ${ROOT}/${jdk_file}; then
+  echo "The blob to be added is identical with the existing one: ${jdk_blob_filename}"
+  git clone java-release java-release-out
+  exit 0
+fi
 
-#
-#if compare_blob_sha_with_new_file ${jdk_blob_filename} ${ROOT}/${jdk_file}; then
-#  bosh add-blob --sha2 --dir java-release jdk/*.tar.gz "$jdk_blob_filename"
-#fi
+bosh add-blob --sha2 --dir java-release jdk/*.tar.gz "$jdk_blob_filename"
 
 if compare_blob_sha_with_new_file ${jre_blob_filename} ${ROOT}/${jre_file}; then
```

Comment:
Do we need this block anymore? As well as adding JRE blob below?

Reply:
Both can be removed. Good spot.